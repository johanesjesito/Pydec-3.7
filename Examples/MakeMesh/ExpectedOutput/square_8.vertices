4
dims=8,2
format=basic
version=1.0
dtype=float64
-0.5 -0.5
-0.5 0.5
-0.03125 0.25
0 -0.5
0 0.5
0.03125 0.25
0.5 -0.5
0.5 0.5
