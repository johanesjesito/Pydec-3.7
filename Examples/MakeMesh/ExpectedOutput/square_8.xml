<?xml version="1.0" ?>
<mesh type="simplicial_mesh">
	<elements>
		<file name="square_8.elements"/>
	</elements>
	<vertices>
		<file name="square_8.vertices"/>
	</vertices>
</mesh>
